### PR TITLE
Fixed a few issues with new test assets

### PR DESCRIPTION
**Pull request by FrederikBolding**
🎉 🎉 🎉

## Description

Fixed a few issues with new test assets as discussed with @blurpesec on Slack.

## Changes

* Fixed contract address comparison
* Renamed some testnet assets/tokens

## Steps to Test

View the dashboard, and see that issues with weird assets are gone! 😄 

## Quality Assurance

- [x] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
- [x] The base branch is develop or gau (no nested branches)
- [x] This is related to a maximum of one Clubhouse story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [x] If code is copied from existing directories, there is an explanation of why this is necesary in the description/changes, and all copying is done in separate commits to make them easy to filter out

### CODE REVIEW

File: common/v2/config/tokens/kovan.json [hunk 1]
```diff
@@ -15,13 +15,13 @@
   },
   {
     "address": "0x4c38cdc08f1260f5c4b21685654393bb1e66a858",
-    "symbol": "DAI",
+    "symbol": "KovanDAI",
```
Comment (vilsbole) [inline common/v2/config/tokens/kovan.json]: @FrederikBolding  @blurpesec  Just want to double check about this.
From what I gather, we change the token symbol to distinguish between each token when we display the totals in Walletbreakdown.
Taking this path would mean we need to create a unique symbol.
And that the conflict will still arise for custom tokens etc.
And we can't get balances for tokens on other networks (not sure this is even possible today).
Shouldn't we modify the `totals` function instead and take the network into account as well ?

Comment (FrederikBolding) [reply common/v2/config/tokens/kovan.json]: Currently we rely too much on the symbols being unique, this is indeed a temporary solution. 

We assume they are unique when looking up rates, giving wrong rates for total balances, but we also assume they are unique when initializing cache here: https://github.com/MyCryptoHQ/MyCrypto/blob/281717a62c8ae71561242b184c65f1c870da9225/common/v2/config/cacheData.ts#L60

If you want, a more permanent solution to at least one of the problems in changing how we initialize the assets into cache?

Comment (FrederikBolding) [reply common/v2/config/tokens/kovan.json]: As discussed with @blurpesec on Slack - having unique symbols for testnet tokens makes it much less confusing for users as well.

**Comment by FrederikBolding** [comment]
Agreed. This is a very temporary solution. Created a card in Clubhouse to figure out a more permanent solution.

**Pull request merged**